### PR TITLE
Details for resource bin packing using RequestedToCapacityRatioResourceAllocationPriority Function

### PR DESCRIPTION
**Pull request by sudeshsh**

### CODE REVIEW

**Comment by dims** [comment]
/milestone v1.16

added on behalf of @sethmccombs ' request on slack

**Comment by jimangel** [comment]
/milestone v1.16

Take 2 😁

**Comment by jimangel** [comment]
/milestone 1.16

Take 3 haha, sorry for the noise!

File: content/en/docs/concepts/configuration/resource-bin-packing.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+               = (100 -((1024-512)*100/1024))
+               = 50
+               = rawScoringFunction(50)
+               = 5
+
+CPU            = resourceScoringFunction((2+1),8)
+               = (100 -((8-3)*100/8))
+               = 37.5
+               = rawScoringFunction(37.5)
+               = 4
```
Comment (ahg-g) [inline content/en/docs/concepts/configuration/resource-bin-packing.md]: 37 will actually be mapped to 3.

Comment (sudeshsh) [reply content/en/docs/concepts/configuration/resource-bin-packing.md]: yea that correct 👍 will update the calculation, thanks for catching it.

File: content/en/docs/concepts/configuration/resource-bin-packing.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+{{< feature-state for_k8s_version="1.16" state="alpha" >}}
+
+The kube-scheduler can be configured to enable bin packing of resources along with extended resources using `RequestedToCapacityRatioResourceAllocation` priority function. Priority functions can be used to fine-tune the kube-scheduler as per custom needs. 
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation Priority Function
```
Comment (simplytunde) [inline content/en/docs/concepts/configuration/resource-bin-packing.md]: is it possible to re-word this to a shorter sentence?

Comment (simplytunde) [reply content/en/docs/concepts/configuration/resource-bin-packing.md]: I am thinking along ```Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation```

Comment (sudeshsh) [reply content/en/docs/concepts/configuration/resource-bin-packing.md]: 👍 I'll update that

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-15990--kubernetes-io-vnext-staging.netlify.com/docs/concepts/configuration/resource-bin-packing/

**Comment by sudeshsh** [comment]
@simplytunde its missing `lgtm` if you could add that

**Comment by zorro786** [comment]
@sudeshsh Can you please update the docs with non deprecated configuration per latest release?

**Comment by sudeshsh** [comment]
@zorro786 He sure I can do that, should I open a new PR?

**Pull request merged**